lib/file_io.rb

```diff
@@ -0,0 +1,9 @@
+class FileIo
+  def self.process_csv(filename, type)
+    elements = []
```

Comment:
I wonder if there will be a refactor warranted here to not require the accumulator. We should leave that to a later time, but if y'all know of something we could use, let's discuss!